src/SwedbankPay.Sdk/Payments/Card/Payment.cs

```diff
@@ -30,26 +32,22 @@ private Payment(PaymentResponse paymentResponse, SwedbankPayHttpClient client)
 
                     case PaymentResourceOperations.DirectAuthorization:
                         operations.DirectAuthorization = async payload =>
-                            await client.SendHttpRequestAndProcessHttpResponse<AuthorizationResponse>(
-                                httpOperation.Request.AttachPayload(payload));
+                            await client.PostAsJsonAsync<AuthorizationResponse>(httpOperation.Href, payload);
```

Comment:
This does not follow the operations design. Here we assume it's a Post instead of using the Method from the Operations response.